### ISSUE TITLE
Lack of working relay code, documentation is outdated.

### ISSUE THREAD

**Issue by kzorel009** [microsoft/CopilotStudioSamples#164]
Hello the pva to bot service connection documentation references outdated code that has been removed. there is some documentation to use co pilot allow list...however allowlist is not available yet. can the team provide some guidance here?

"Allowlist is coming soon"

**Comment by HenryJammes** [comment]
Hi @kzorel009, indeed "PVA as a skill" (https://learn.microsoft.com/en-us/microsoft-copilot-studio/advanced-use-pva-as-a-skill) is only available in the classic version. Work is in progress to make it available in the latest version of Copilot Studio. Let me see if I can get you an ETA.

**Comment by it360mz** [comment]
This is effecting us too, can you provide an ETA

**Comment by it360inc** [comment]
I will pile on here.   This is keeping us from moving forward.

**Comment by HenryJammes** [comment]
@it360mz  @it360inc can you both confirm that the ask is for 'Copilot Studio unified authoring - i.e. PVA 2 - as a skill', like PVA as a skill in the classic version? (i.e. this: https://learn.microsoft.com/en-us/microsoft-copilot-studio/advanced-use-pva-as-a-skill)

**Comment by it360mz** [comment]
Yes we're following https://learn.microsoft.com/en-us/microsoft-copilot-studio/advanced-use-pva-as-a-skill#add-your-bot-framework-bot-to-the-allowlist-for-your-microsoft-copilot-studio-bot at step 3 where that links and getting the Allowlist is coming soon at that step

**Comment by HenryJammes** [comment]
Thank you @it360mz, I'm hoping this could be a better of weeks before we enable this. I'll keep you posted. If it's urgent, send me a message so we can follow up?
